app/utils/WkConf.scala

```diff
@@ -60,6 +60,13 @@ class WkConf @Inject()(configuration: Configuration) extends ConfigReader with L
       val children = List(User)
     }
 
+    object OIDC {
```

Comment:
```suggestion
    object SingleSignOn {
      object OpenIdConnect {
```

It’s a bit arbitrary but I’d move this out of the webKnossos config block and into the top level, since it is not a feature that only makes sense in the webKnossos context. Instead, I’d wrap it in SingleSignOn, since we may add more providers in the future.

Reply:
sure